tests/tests_api_v1_0/test_filters.py

```diff
@@ -334,9 +333,24 @@ def tearDown(self):
 
     def test_filter_range_date_and_datetime_fields(self):
         start_date = self.date.add(days=-1).to_date_string()
-        start_datetime = format_datetime(self.date.add(days=-1))
+        start_datetime = "2017-10-27T00:00:00Z"
+        print(start_datetime)
         end_date = self.date.add(days=3).to_date_string()
-        end_datetime = format_datetime(self.date.add(days=3))
+        end_datetime = "2017-10-27T00:00:00Z"
+        url_date = '/api/v1/date-utc/?date__range={},{}&datetime__range={},{}'.format(
```

Comment:
It's better to separate the tests and have one with the field `date` and another one with the field `datetime` only.  
You can still have a test with both after.